SpatialGDKEditor/Source/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#include "SpatialGDKEditorSettings.h"
+
+USpatialGDKEditorSettings::USpatialGDKEditorSettings(const FObjectInitializer& ObjectInitializer)
+	: Super(ObjectInitializer)
+	, bGenerateSchemaForAllSupportedClasses(true)
+{
+	SpatialOSDirectory.Path = TEXT("");
+	GeneratedSchemaOutputFolder.Path = TEXT("");
+}
+
+FString USpatialGDKEditorSettings::ToString()
+{
+	TArray<FStringFormatArg> Args;
+	Args.Add(SpatialOSDirectory.Path);
+	Args.Add(SpatialOSSnapshotPath.Path);
+	Args.Add(GeneratedSchemaOutputFolder.Path);
+	Args.Add(bGenerateSchemaForAllSupportedClasses);
```

Comment:
With regards to Valentyn's comments, this also has gone away with Mike's changes.